bin/docker-entrypoint

```diff
@@ -50,7 +50,7 @@ server() {
 }
 
 create_db() {
-  exec /app/manage.py database create_tables
+  exec /app/manage.py database create-tables
```

Comment:
That's a strange change.  Was this PR created using GitHub Copilot or similar?

Reply:
After the release of [click 7.0](https://github.com/pallets/click/issues/1123), it disrupted the command name. Maybe we should compromise?

Reply:
I **think** it needs to match the function name that gets called:

https://github.com/getredash/redash/blob/bbd0a21831d04562c769e4b6b3693c062b146654/redash/cli/database.py#L45

The other python function names use underscores so we should probably keep with that I guess. :smile: